rules/windows/credential_access_remote_sam_secretsdump.toml

```
@@ -30,16 +30,17 @@ credentials to access other systems in the domain.
 
 #### Possible investigation steps
 
-- Identify the target host role, involved account, and source host.
-- Determine the privileges assigned to any compromised accounts.
-- Investigate other alerts related to the involved user and source host in the last 48 hours.
-- Scope potentially compromised accounts. Analysts can do this by searching for login events (for example, 4624) to the target
+- Identify the specifics of the involved asset(s), as its role, criticality, as well as the user(s) associated with it.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Determine the privileges of the compromised accounts.
+- Investigate other alerts associated with the user/source host during the past 48 hours.
+- Scope potentially compromised accounts. Analysts can do this by searching for login events (e.g., 4624) to the target
```

"Scope" seems unclear and jargony here; maybe a more straightforward verb would help:
```suggestion
- Investigate potentially compromised accounts. Analysts can do this by searching for login events (e.g., 4624) to the target
```